src/run-on-arch.js

```diff
@@ -9,6 +9,19 @@ function slug(str) {
   return str.replace(/[^a-zA-Z0-9]/g, '-').toLowerCase();
 }
 
+function docker_arch(qemu_arch) {
```

Comment:
For consistency, would be good to make this `dockerArch` and `qemuArch`.

Reply:
Oh, right, JS cares about cases in different ways than Bash.